### PR TITLE
fix: Support building golang packages again

### PR DESCRIPTION
**Pull request by oleorhagen**
Re-enable building of Golang packages with our new cross-compiler setup.

Ticket: MEN-6759
Changelog: Title

### CODE REVIEW

File: mender-dist-packages-building/install-armv6-cross-toolchain [hunk 1]
```diff
@@ -238,7 +238,14 @@ function install_packages_to_toolchain_root() {
...
         echo "Extracting ${pkg} to an archive (tar.(xz|gz))...";
         ar -x ${cache}/${pkg}_*_armhf.deb data.tar.xz;
+        # List the file contents
+        tar tvf data.tar.xz
+        # Since '/arm-linux-gnueabihf/' is not in the default LIBRARY_PATH for
+        # the compiler, simply remove it
         tar --extract  --directory=${toolchain_sysroot} --file=data.tar.xz --transform='s,/arm-linux-gnueabihf,,'
+        # And also keep the same files, with the 'arm-linux-gnueabihf' path to
+        # keep 'pkg-config' happy
+        tar --extract  --directory=${toolchain_sysroot} --file=data.tar.xz
```
Comment (lluiscampos) [inline mender-dist-packages-building/install-armv6-cross-toolchain:248]: Is this all it takes? :tada:

**Review by lluiscampos — APPROVED** [review]
I have no way to verify myself that these changes are enough. The changes look good and I trust your testing :+1:

File: mender-dist-packages-building/install-armv6-cross-toolchain (hunk 1 above)
Comment (oleorhagen) [reply mender-dist-packages-building/install-armv6-cross-toolchain:248]: This, alongside re-installing `pkg-config` :)

Comment (lluiscampos) [reply mender-dist-packages-building/install-armv6-cross-toolchain:248]: (That was my point, that this is all i takes for`pkg-config` to work properly :+1: )

Comment (oleorhagen) [reply mender-dist-packages-building/install-armv6-cross-toolchain:248]: haha, yup :smiley_cat:

**Pull request merged**